rules/windows/builtin/application/screenconnect/win_app_remote_binary_execution.yaml

```diff
@@ -0,0 +1,24 @@
+title: Remote Access Tool - Remote Binary Execution
+id: 5d19eb78-5b5b-4ef2-a9f0-4bfa94d58a13
+related:
+    - id: b1f73849-6329-4069-bc8f-78a604bb8b23
+      type: similar
+status: experimental
+description: ScreenConnect RMM has feature to remotely execute binaries on a target machine.
+author: Ali Alwashali
+date: 2023/10/10
+tags:
+    - attack.execution
+    - attack.t1059.003
+logsource:
+    service: application
+    product: windows
+detection:
+    selection:
+        Provider_Name: 'ScreenConnect'
+        EventID: 201
+        Data|contains: 'Transferred files with action'
```

Comment:
Does this log contain the size or type of files transferred? Do you have an example?

Reply:
Unfortunately no, it's just one line

![image](https://github.com/SigmaHQ/sigma/assets/22593441/fac80622-2bcc-4f44-89f7-787433946e03)